.gitignore

```diff
@@ -1,11 +1,94 @@
-target
-bin
-.classpath
-.settings
-.project
+### Linux
```

Comment:
Of course this is templates, because this files can be accidentally committed by some one